src/gameObjects/map_start/treasureChest.gameObject.js

```diff
@@ -0,0 +1,72 @@
+import { scaleFactor } from '../../constants';
+
+export const treasureChest = (k, map, spawnpoints) => {
+    // Load the sprite for the Open treasure chest
+    k.loadSprite(
+        'treasureChestClosed',
+        './assets/sprites/treasure_chest_closed.png',
+        {
+            sliceX: 1,
+            sliceY: 1,
+            anims: {
+                idle: 0,
+            },
+        }
+    );
+    // Load the sprite for the Closed treasure chest open
+    k.loadSprite(
+        'treasureChestOpen',
+        './assets/sprites/treasure_chest_open.png',
+        {
+            sliceX: 1,
+            sliceY: 1,
+            anims: {
+                idle: 0,
+            },
+        }
+    );
+
+    // Load the chest opening sound
+    k.loadSound('chestOpen', './assets/sounds/chest_opening.wav');
+
+    const chestPosition = spawnpoints.treasureChest;
```

Comment:
We don't have a spawnpoint for a treasure chest right now. Could you figure out a appropriate hardcoded position for the treasure chest please🙏🏻?